packages/falso/src/lib/programming-language.ts

```diff
@@ -0,0 +1,35 @@
+import { rand } from './core';
+
+export function programmingLanguage() {
+  return rand([
```

Comment:
Can you please migrate to the new way using a JSON file?

Reply:
@NetanelBasal PR updated thanks!